src/sentry/integrations/vsts/integration.py

```diff
@@ -128,26 +132,27 @@ def get_pipeline_views(self):
 
         return [
             identity_pipeline_view,
+            AccountConfigView(),
         ]
 
     def build_integration(self, state):
         data = state['identity']['data']
-        account = state['identity']['account']
-        instance = state['identity']['instance']
-
+        account = state['account']
+        instance = state['instance']
+        user = get_user_info(data['access_token'])
         scopes = sorted(VSTSIdentityProvider.oauth_scopes)
+
         return {
             'name': account['AccountName'],
             'external_id': account['AccountId'],
             'metadata': {
                 'domain_name': instance,
                 'scopes': scopes,
```

Comment:
So, should the scopes still be in the integration? Are we putting it in both places now? I think if we're putting the scopes in the user identity, we should take it out of the integration.

Reply:
yea i'll take it out, just wasn't sure if you need it on the integration side for anything

Reply:
take it out of what part @adhiraj ? Feels like scopes should be part of the users identity.

Reply:
out of the integration metadata